features/support/driver.js

```diff
@@ -166,9 +166,15 @@ class Driver {
     return browser.quit()
   }
 
+  async getCurrentUrlName () {
+    const browser = this.browser
+    const url1 = await browser.getCurrentUrl()
+    return url1
+  }
+
   takeScreenshots1 (filename) {
```

Comment:
Just wondering why you called this **takeScreenshots1** instead of **takeScreenshots**

Reply:
returning await browser.getCurrentUrl()